michelson.md

```diff
@@ -1711,10 +1713,17 @@ value is invalid.
 
   syntax FailedStack ::= #FailureFromMutezValue(Mutez, Int, Int) [function]
   // ----------------------------------------------------------------------
-  rule #FailureFromMutezValue(#Mutez(I), I1, I2)
-    => ( MutezOverflow I1 I2 ) requires I >=Int #MutezOverflowLimit
-  rule #FailureFromMutezValue(#Mutez(I), I1, I2)
-    => ( MutezUnderflow I1 I2 ) requires I <Int 0
+  rule #FailureFromMutezValue(#Mutez(I), I1, I2) => ( MutezOverflow I1 I2 )  requires I >=Int #MutezOverflowLimit
+  rule #FailureFromMutezValue(#Mutez(I), I1, I2) => ( MutezUnderflow I1 I2 ) requires I  <Int 0
+```
+
+```symbolic
+  rule #FailureFromMutezValue(#Mutez(I), I1, I2) => ( MutezOverflow I1 I2 )  requires I >=Int #MutezOverflowLimit [simplification]
+  rule #FailureFromMutezValue(#Mutez(I), I1, I2) => ( MutezUnderflow I1 I2 ) requires I  <Int 0                   [simplification]
+
+  rule #Ceil(#FailureFromMutezValue(#Mutez(I), _I1, _I2)) => #Bottom
+    requires notBool(I >=Int #MutezOverflowLimit orBool I  <Int 0)
+    [simplification]
```

Comment:
```suggestion
    [anywhere, simplification]
```

Reply:
What does the `anywhere` do in this context? I've generally seen anywhere over program constructs, and not things like `#Ceil`.